lib/modules/manager/bundler/artifacts.spec.ts

```diff
@@ -967,6 +967,28 @@ describe('modules/manager/bundler/artifacts', () => {
         expect(res).toMatchObject([{ file: { path: 'Gemfile.lock' } }]);
         expect(execSnapshots).toMatchObject([{ cmd: 'bundler lock' }]);
       });
+
+      it('updates the Gemfile.lock when bundler upgraded', async () => {
```

Comment:
The test description needs a small rewrite. Would one of these work?

- updates the Gemfile.lock when updating the bundler package manager
- updates the Gemfile.lock after updating the bundler package manager

Reply:
I'm not sure I understand what extra information is conveyed by either of those. Anyone reading the source code of the bundler manager knows that bundler is a package manager already.

Reply:
> I'm not sure I understand what extra information is conveyed by either of those.

I thought it was nice to mention that bundler is the package manager. But if the ones reading the code already know about it, you can leave it out. :smile: 

My point is that the tenses don't match in the description. Current situation:

- `updates` is current tense ("now")
- `upgraded` is past tense ("the past")

So it seems better to be clear about _what_ happens _when_ with a small rewrite with a connecting word like `when/after` and using the current tense. I've dropped the "package manager" part for you already. :upside_down_face: 

- updates the Gemfile.lock when updating bundler
- updates the Gemfile.lock after updating bundler